From 98551cf4a264a69366e5b50044b33f25d1f4807e Mon Sep 17 00:00:00 2001
From: Michael Achenbach <machenbach@chromium.org>
Date: Sun, 6 Nov 2022 14:51:24 +0100
Subject: [PATCH] [infra] Update configs to compilator/orchestrator names

This used the production builder names in MB and test configs
after the orchestrator roll-out.

This can first land after:
https://crrev.com/c/4002978

Bug: chromium:890222
Change-Id: I7215fdb86e7ddddac5c2d3172fb112b30a373141
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4006615
Reviewed-by: Liviu Rau <liviurau@google.com>
Reviewed-by: Alexander Schulze <alexschulze@chromium.org>
Commit-Queue: Michael Achenbach <machenbach@chromium.org>
Cr-Commit-Position: refs/heads/main@{#84090}
---
 infra/mb/mb_config.pyl     | 142 ++++++++++++++++++-------------------
 infra/testing/builders.pyl | 140 ++++++++++++++++++------------------
 2 files changed, 141 insertions(+), 141 deletions(-)

diff --git a/infra/mb/mb_config.pyl b/infra/mb/mb_config.pyl
index f336359462..b5d6231600 100644
--- a/infra/mb/mb_config.pyl
+++ b/infra/mb/mb_config.pyl
@@ -208,98 +208,98 @@
     'tryserver.v8': {
       'v8_android_arm_compile_rel': 'release_android_arm',
       'v8_android_arm64_compile_dbg': 'debug_android_arm64',
-      'v8_android_arm64_n5x_rel_ng': 'release_android_arm64',
-      'v8_fuchsia_rel_ng': 'release_x64_fuchsia_trybot',
+      'v8_android_arm64_n5x_compile_rel': 'release_android_arm64',
+      'v8_fuchsia_compile_rel': 'release_x64_fuchsia_trybot',
       'v8_ios_simulator': 'release_x64_ios_simulator',
-      'v8_linux_rel_ng': 'release_x86_gcmole_trybot',
-      'v8_linux_optional_rel_ng': 'release_x86_trybot',
-      'v8_linux_verify_csa_rel_ng': 'release_x86_verify_csa',
-      'v8_linux_nodcheck_rel_ng': 'release_x86_minimal_symbols',
-      'v8_linux_dbg_ng': 'debug_x86_trybot',
+      'v8_linux_compile_rel': 'release_x86_gcmole_trybot',
+      'v8_linux_optional_compile_rel': 'release_x86_trybot',
+      'v8_linux_verify_csa_compile_rel': 'release_x86_verify_csa',
+      'v8_linux_nodcheck_compile_rel': 'release_x86_minimal_symbols',
+      'v8_linux_compile_dbg': 'debug_x86_trybot',
       'v8_linux_noi18n_compile_dbg': 'debug_x86_no_i18n',
-      'v8_linux_noi18n_rel_ng': 'release_x86_no_i18n_trybot',
-      'v8_linux_gc_stress_dbg_ng': 'debug_x86_trybot',
+      'v8_linux_noi18n_compile_rel': 'release_x86_no_i18n_trybot',
+      'v8_linux_gc_stress_compile_dbg': 'debug_x86_trybot',
       'v8_linux_shared_compile_rel': 'release_x86_shared_verify_heap',
       'v8_linux_vtunejit': 'debug_x86_vtunejit',
-      'v8_linux64_arm64_no_pointer_compression_rel_ng':
+      'v8_linux64_arm64_no_pointer_compression_compile_rel':
           'release_simulate_arm64_no_pointer_compression',
-      'v8_linux64_cppgc_non_default_dbg_ng': 'debug_x64_non_default_cppgc',
-      'v8_linux64_dbg_ng': 'debug_x64_trybot',
-      'v8_linux64_no_sandbox_dbg_ng': 'debug_x64_no_sandbox',
-      'v8_linux64_dict_tracking_dbg_ng': 'debug_x64_dict_tracking_trybot',
-      'v8_linux64_disable_runtime_call_stats_rel_ng': 'release_x64_disable_runtime_call_stats',
-      'v8_linux64_external_code_space_dbg_ng': 'debug_x64_external_code_space',
+      'v8_linux64_cppgc_non_default_compile_dbg': 'debug_x64_non_default_cppgc',
+      'v8_linux64_compile_dbg': 'debug_x64_trybot',
+      'v8_linux64_no_sandbox_compile_dbg': 'debug_x64_no_sandbox',
+      'v8_linux64_dict_tracking_compile_dbg': 'debug_x64_dict_tracking_trybot',
+      'v8_linux64_disable_runtime_call_stats_compile_rel': 'release_x64_disable_runtime_call_stats',
+      'v8_linux64_external_code_space_compile_dbg': 'debug_x64_external_code_space',
       'v8_linux64_css_compile_dbg': 'debug_x64_conservative_stack_scanning',
-      'v8_linux64_gc_stress_custom_snapshot_dbg_ng': 'debug_x64_trybot_custom',
-      'v8_linux64_gc_stress_dbg_ng': 'debug_x64_trybot',
+      'v8_linux64_gc_stress_custom_snapshot_compile_dbg': 'debug_x64_trybot_custom',
+      'v8_linux64_gc_stress_compile_dbg': 'debug_x64_trybot',
       'v8_linux64_gcc_compile_dbg': 'debug_x64_gcc',
       'v8_linux64_gcc_light_compile_dbg': 'debug_x64_gcc',
-      'v8_linux64_gcc_rel_ng': 'release_x64_gcc',
+      'v8_linux64_gcc_compile_rel': 'release_x64_gcc',
       'v8_linux64_gcov_coverage': 'release_x64_gcc_coverage',
       'v8_linux64_header_includes_dbg': 'debug_x64_header_includes',
-      'v8_linux64_heap_sandbox_dbg_ng': 'debug_x64_heap_sandbox',
-      'v8_linux64_minor_mc_dbg_ng': 'debug_x64_trybot',
-      'v8_linux_arm64_sim_heap_sandbox_dbg_ng': 'debug_x64_heap_sandbox_arm64_sim',
-      'v8_linux64_fyi_rel_ng': 'release_x64_test_features_trybot',
-      'v8_linux64_nodcheck_rel_ng': 'release_x64',
-      'v8_linux64_perfetto_dbg_ng': 'debug_x64_perfetto',
-      'v8_linux64_no_pointer_compression_rel_ng': 'release_x64_no_pointer_compression',
-      'v8_linux64_rel_ng': 'release_x64_test_features_trybot',
-      'v8_linux64_no_sandbox_rel_ng': 'release_x64_no_sandbox',
-      'v8_linux64_predictable_rel_ng': 'release_x64_predictable',
+      'v8_linux64_heap_sandbox_compile_dbg': 'debug_x64_heap_sandbox',
+      'v8_linux64_minor_mc_compile_dbg': 'debug_x64_trybot',
+      'v8_linux_arm64_sim_heap_sandbox_compile_dbg': 'debug_x64_heap_sandbox_arm64_sim',
+      'v8_linux64_fyi_compile_rel': 'release_x64_test_features_trybot',
+      'v8_linux64_nodcheck_compile_rel': 'release_x64',
+      'v8_linux64_perfetto_compile_dbg': 'debug_x64_perfetto',
+      'v8_linux64_no_pointer_compression_compile_rel': 'release_x64_no_pointer_compression',
+      'v8_linux64_compile_rel': 'release_x64_test_features_trybot',
+      'v8_linux64_no_sandbox_compile_rel': 'release_x64_no_sandbox',
+      'v8_linux64_predictable_compile_rel': 'release_x64_predictable',
       'v8_linux64_shared_compile_rel': 'release_x64_shared_verify_heap',
-      'v8_linux64_single_generation_dbg_ng': 'debug_x64_single_generation',
+      'v8_linux64_single_generation_compile_dbg': 'debug_x64_single_generation',
       'v8_linux64_no_wasm_compile_rel': 'release_x64_webassembly_disabled',
-      'v8_linux64_verify_csa_rel_ng': 'release_x64_verify_csa',
-      'v8_linux64_asan_rel_ng': 'release_x64_asan_minimal_symbols',
-      'v8_linux64_cfi_rel_ng': 'release_x64_cfi',
-      'v8_linux64_fuzzilli_rel_ng': 'release_x64_fuzzilli',
-      'v8_linux64_loong64_rel_ng': 'release_simulate_loong64',
-      'v8_linux64_msan_rel_ng': 'release_simulate_arm64_msan_minimal_symbols',
-      'v8_linux_riscv32_rel_ng': 'release_simulate_riscv32',
-      'v8_linux64_riscv64_rel_ng': 'release_simulate_riscv64',
-      'v8_linux64_tsan_rel_ng': 'release_x64_tsan_minimal_symbols',
-      'v8_linux64_tsan_no_cm_rel_ng': 'release_x64_tsan_no_cm',
-      'v8_linux64_tsan_isolates_rel_ng':
+      'v8_linux64_verify_csa_compile_rel': 'release_x64_verify_csa',
+      'v8_linux64_asan_compile_rel': 'release_x64_asan_minimal_symbols',
+      'v8_linux64_cfi_compile_rel': 'release_x64_cfi',
+      'v8_linux64_fuzzilli_compile_rel': 'release_x64_fuzzilli',
+      'v8_linux64_loong64_compile_rel': 'release_simulate_loong64',
+      'v8_linux64_msan_compile_rel': 'release_simulate_arm64_msan_minimal_symbols',
+      'v8_linux_riscv32_compile_rel': 'release_simulate_riscv32',
+      'v8_linux64_riscv64_compile_rel': 'release_simulate_riscv64',
+      'v8_linux64_tsan_compile_rel': 'release_x64_tsan_minimal_symbols',
+      'v8_linux64_tsan_no_cm_compile_rel': 'release_x64_tsan_no_cm',
+      'v8_linux64_tsan_isolates_compile_rel':
           'release_x64_tsan_minimal_symbols',
-      'v8_linux64_ubsan_rel_ng': 'release_x64_ubsan_minimal_symbols',
-      'v8_odroid_arm_rel_ng': 'release_arm',
+      'v8_linux64_ubsan_compile_rel': 'release_x64_ubsan_minimal_symbols',
+      'v8_odroid_arm_compile_rel': 'release_arm',
       'v8_linux_torque_compare': 'torque_compare',
       # TODO(machenbach): Remove after switching to x64 on infra side.
-      'v8_win_dbg_ng': 'debug_x86_trybot',
-      'v8_win_rel_ng': 'release_x86_trybot',
-      'v8_win64_asan_rel_ng': 'release_x64_asan_no_lsan',
+      'v8_win_compile_dbg': 'debug_x86_trybot',
+      'v8_win_compile_rel': 'release_x86_trybot',
+      'v8_win64_asan_compile_rel': 'release_x64_asan_no_lsan',
       'v8_win64_msvc_light_compile_rel': 'release_x64_msvc',
-      'v8_win64_dbg_ng': 'debug_x64_minimal_symbols',
-      'v8_win64_msvc_rel_ng': 'release_x64_msvc',
-      'v8_win64_rel_ng': 'release_x64_trybot',
-      'v8_mac_arm64_rel_ng': 'release_arm64',
-      'v8_mac_arm64_dbg_ng': 'debug_arm64',
-      'v8_mac_arm64_full_dbg_ng': 'full_debug_arm64',
-      'v8_mac_arm64_no_pointer_compression_dbg_ng': 'debug_arm64_no_pointer_compression',
-      'v8_mac_arm64_sim_rel_ng': 'release_simulate_arm64_trybot',
-      'v8_mac_arm64_sim_dbg_ng': 'debug_simulate_arm64',
-      'v8_mac_arm64_sim_nodcheck_rel_ng': 'release_simulate_arm64',
-      'v8_mac64_gc_stress_dbg_ng': 'debug_x64_trybot',
-      'v8_mac64_rel_ng': 'release_x64_trybot',
+      'v8_win64_compile_dbg': 'debug_x64_minimal_symbols',
+      'v8_win64_msvc_compile_rel': 'release_x64_msvc',
+      'v8_win64_compile_rel': 'release_x64_trybot',
+      'v8_mac_arm64_compile_rel': 'release_arm64',
+      'v8_mac_arm64_compile_dbg': 'debug_arm64',
+      'v8_mac_arm64_full_compile_dbg': 'full_debug_arm64',
+      'v8_mac_arm64_no_pointer_compression_compile_dbg': 'debug_arm64_no_pointer_compression',
+      'v8_mac_arm64_sim_compile_rel': 'release_simulate_arm64_trybot',
+      'v8_mac_arm64_sim_compile_dbg': 'debug_simulate_arm64',
+      'v8_mac_arm64_sim_nodcheck_compile_rel': 'release_simulate_arm64',
+      'v8_mac64_gc_stress_compile_dbg': 'debug_x64_trybot',
+      'v8_mac64_compile_rel': 'release_x64_trybot',
       'v8_mac64_dbg': 'debug_x64',
-      'v8_mac64_dbg_ng': 'debug_x64',
-      'v8_mac64_compile_full_dbg_ng': 'full_debug_x64',
-      'v8_mac64_asan_rel_ng': 'release_x64_asan_no_lsan',
-      'v8_linux_arm_rel_ng': 'release_simulate_arm_trybot',
+      'v8_mac64_compile_dbg': 'debug_x64',
+      'v8_mac64_compile_full_compile_dbg': 'full_debug_x64',
+      'v8_mac64_asan_compile_rel': 'release_x64_asan_no_lsan',
+      'v8_linux_arm_compile_rel': 'release_simulate_arm_trybot',
       'v8_linux_arm_lite_compile_dbg': 'debug_simulate_arm_lite',
-      'v8_linux_arm_lite_rel_ng': 'release_simulate_arm_lite_trybot',
-      'v8_linux_arm_dbg_ng': 'debug_simulate_arm',
+      'v8_linux_arm_lite_compile_rel': 'release_simulate_arm_lite_trybot',
+      'v8_linux_arm_compile_dbg': 'debug_simulate_arm',
       'v8_linux_arm_armv8a_rel': 'release_simulate_arm_trybot',
       'v8_linux_arm_armv8a_dbg': 'debug_simulate_arm',
-      'v8_linux_arm64_rel_ng': 'release_simulate_arm64_trybot',
-      'v8_linux_arm64_cfi_rel_ng' : 'release_simulate_arm64_cfi',
-      'v8_linux_arm64_dbg_ng': 'debug_simulate_arm64',
-      'v8_linux_arm64_gc_stress_dbg_ng': 'debug_simulate_arm64',
+      'v8_linux_arm64_compile_rel': 'release_simulate_arm64_trybot',
+      'v8_linux_arm64_cfi_compile_rel' : 'release_simulate_arm64_cfi',
+      'v8_linux_arm64_compile_dbg': 'debug_simulate_arm64',
+      'v8_linux_arm64_gc_stress_compile_dbg': 'debug_simulate_arm64',
       'v8_linux_mips64el_compile_rel': 'release_simulate_mips64el',
-      'v8_numfuzz_rel_ng': 'release_x64',
-      'v8_numfuzz_dbg_ng': 'debug_x64',
-      'v8_numfuzz_tsan_rel_ng': 'release_x64_tsan',
+      'v8_numfuzz_compile_rel': 'release_x64',
+      'v8_numfuzz_compile_dbg': 'debug_x64',
+      'v8_numfuzz_tsan_compile_rel': 'release_x64_tsan',
     },
   },
 
diff --git a/infra/testing/builders.pyl b/infra/testing/builders.pyl
index 48a3cbb3b0..19d5e18452 100644
--- a/infra/testing/builders.pyl
+++ b/infra/testing/builders.pyl
@@ -32,7 +32,7 @@
   ### luci.v8.try
   ##############################################################################
   # Android
-  'v8_android_arm64_n5x_rel_ng_triggered': {
+  'v8_android_arm64_n5x_rel': {
     'swarming_dimensions' : {
       'device_os': 'MMB29Q',
       'device_type': 'bullhead',
@@ -46,7 +46,7 @@
   },
   ##############################################################################
   # Fuchsia
-  'v8_fuchsia_rel_ng_triggered': {
+  'v8_fuchsia_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -64,7 +64,7 @@
   },
   ##############################################################################
   # Linux32
-  'v8_linux_dbg_ng_triggered': {
+  'v8_linux_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -109,7 +109,7 @@
       },
     ],
   },
-  'v8_linux_gc_stress_dbg_ng_triggered': {
+  'v8_linux_gc_stress_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -118,7 +118,7 @@
       {'name': 'd8testing', 'test_args': ['--gc-stress'], 'shards': 5},
     ],
   },
-  'v8_linux_nodcheck_rel_ng_triggered': {
+  'v8_linux_nodcheck_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -143,7 +143,7 @@
       },
     ],
   },
-  'v8_linux_noi18n_rel_ng_triggered': {
+  'v8_linux_noi18n_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -153,7 +153,7 @@
       {'name': 'v8testing', 'variant': 'default', 'shards': 2},
     ],
   },
-  'v8_linux_rel_ng_triggered': {
+  'v8_linux_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -182,7 +182,7 @@
       {'name': 'gcmole'},
     ],
   },
-  'v8_linux_optional_rel_ng_triggered': {
+  'v8_linux_optional_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -285,7 +285,7 @@
       },
     ],
   },
-  'v8_linux_verify_csa_rel_ng_triggered': {
+  'v8_linux_verify_csa_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -295,7 +295,7 @@
   },
   ##############################################################################
   # Linux32 with arm simulators
-  'v8_linux_arm_dbg_ng_triggered': {
+  'v8_linux_arm_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -307,7 +307,7 @@
       {'name': 'v8testing', 'variant': 'extra', 'shards': 10},
     ],
   },
-  'v8_linux_arm_lite_rel_ng_triggered': {
+  'v8_linux_arm_lite_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -315,7 +315,7 @@
       {'name': 'v8testing', 'variant': 'default', 'shards': 4},
     ],
   },
-  'v8_linux_arm_rel_ng_triggered': {
+  'v8_linux_arm_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -329,7 +329,7 @@
   },
   ##############################################################################
   # Linux64
-  'v8_linux64_asan_rel_ng_triggered': {
+  'v8_linux64_asan_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -340,7 +340,7 @@
       {'name': 'v8testing', 'variant': 'slow_path'},
     ],
   },
-  'v8_linux64_cfi_rel_ng_triggered': {
+  'v8_linux64_cfi_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -352,7 +352,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_cppgc_non_default_dbg_ng_triggered': {
+  'v8_linux64_cppgc_non_default_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -361,7 +361,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_dbg_ng_triggered': {
+  'v8_linux64_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -385,7 +385,7 @@
       {'name': 'mjsunit', 'variant': 'maglev'},
     ],
   },
-  'v8_linux64_dict_tracking_dbg_ng_triggered': {
+  'v8_linux64_dict_tracking_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -394,7 +394,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_disable_runtime_call_stats_rel_ng_triggered': {
+  'v8_linux64_disable_runtime_call_stats_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -402,7 +402,7 @@
       {'name': 'v8testing'},
     ],
   },
-  'v8_linux64_external_code_space_dbg_ng_triggered': {
+  'v8_linux64_external_code_space_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -411,14 +411,14 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_fuzzilli_rel_ng_triggered': {
+  'v8_linux64_fuzzilli_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
     # TODO(almuthanna): Add a new test config for the fuzzilli suite.
     'tests': [],
   },
-  'v8_linux64_fyi_rel_ng_triggered': {
+  'v8_linux64_fyi_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -440,7 +440,7 @@
       {'name': 'mjsunit', 'variant': 'wasm_write_protect_code'},
     ],
   },
-  'v8_linux64_gc_stress_custom_snapshot_dbg_ng_triggered': {
+  'v8_linux64_gc_stress_custom_snapshot_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -452,7 +452,7 @@
       },
     ],
   },
-  'v8_linux64_gc_stress_dbg_ng_triggered': {
+  'v8_linux64_gc_stress_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -473,7 +473,7 @@
       },
     ],
   },
-  'v8_linux64_gcc_rel_ng_triggered': {
+  'v8_linux64_gcc_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-20.04',
     },
@@ -489,7 +489,7 @@
       {'name': 'v8testing'},
     ],
   },
-  'v8_linux64_heap_sandbox_dbg_ng_triggered': {
+  'v8_linux64_heap_sandbox_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -497,7 +497,7 @@
       {'name': 'v8testing', 'shards': 4},
     ],
   },
-  'v8_linux64_minor_mc_dbg_ng_triggered': {
+  'v8_linux64_minor_mc_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -510,7 +510,7 @@
       {'name': 'mjsunit', 'variant': 'minor_mc'},
     ],
   },
-  'v8_linux64_msan_rel_ng_triggered': {
+  'v8_linux64_msan_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -519,7 +519,7 @@
       {'name': 'v8testing', 'shards': 5},
     ],
   },
-  'v8_linux64_nodcheck_rel_ng_triggered': {
+  'v8_linux64_nodcheck_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -545,7 +545,7 @@
       {'name': 'mjsunit', 'variant': 'maglev'},
     ],
   },
-  'v8_linux64_perfetto_dbg_ng_triggered': {
+  'v8_linux64_perfetto_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -553,7 +553,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_no_pointer_compression_rel_ng_triggered': {
+  'v8_linux64_no_pointer_compression_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -561,7 +561,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_no_sandbox_dbg_ng_triggered': {
+  'v8_linux64_no_sandbox_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -570,7 +570,7 @@
       {'name': 'v8testing', 'shards': 5},
     ],
   },
-  'v8_linux64_single_generation_dbg_ng_triggered': {
+  'v8_linux64_single_generation_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -578,7 +578,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_rel_ng_triggered': {
+  'v8_linux64_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -601,7 +601,7 @@
       {'name': 'mjsunit', 'variant': 'maglev'},
     ],
   },
-  'v8_linux64_predictable_rel_ng_triggered': {
+  'v8_linux64_predictable_rel': {
     'swarming_dimensions': {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -612,7 +612,7 @@
       {'name': 'mozilla'},
     ],
   },
-  'v8_linux64_no_sandbox_rel_ng_triggered': {
+  'v8_linux64_no_sandbox_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64-avx2',
       'os': 'Ubuntu-18.04',
@@ -621,7 +621,7 @@
       {'name': 'v8testing', 'shards': 2},
     ],
   },
-  'v8_linux64_tsan_rel_ng_triggered': {
+  'v8_linux64_tsan_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -635,7 +635,7 @@
       {'name': 'v8testing', 'variant': 'stress_concurrent_allocation', 'shards': 2},
     ],
   },
-  'v8_linux64_tsan_no_cm_rel_ng_triggered': {
+  'v8_linux64_tsan_no_cm_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -653,7 +653,7 @@
       {'name': 'v8testing', 'variant': 'stress_concurrent_inlining', 'shards': 2},
     ],
   },
-  'v8_linux64_tsan_isolates_rel_ng_triggered': {
+  'v8_linux64_tsan_isolates_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -661,7 +661,7 @@
       {'name': 'v8testing', 'test_args': ['--isolates'], 'shards': 7},
     ],
   },
-  'v8_linux64_ubsan_rel_ng_triggered': {
+  'v8_linux64_ubsan_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -669,7 +669,7 @@
       {'name': 'v8testing', 'shards': 2},
     ],
   },
-  'v8_linux64_verify_csa_rel_ng_triggered': {
+  'v8_linux64_verify_csa_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -679,7 +679,7 @@
   },
   ##############################################################################
   # Linux64 with arm64 simulators
-  'v8_linux_arm64_dbg_ng_triggered': {
+  'v8_linux_arm64_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -691,7 +691,7 @@
       {'name': 'v8testing', 'variant': 'extra', 'shards': 14},
     ],
   },
-  'v8_linux_arm64_gc_stress_dbg_ng_triggered': {
+  'v8_linux_arm64_gc_stress_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -699,7 +699,7 @@
       {'name': 'd8testing', 'test_args': ['--gc-stress'], 'shards': 12},
     ],
   },
-  'v8_linux_arm64_sim_heap_sandbox_dbg_ng_triggered': {
+  'v8_linux_arm64_sim_heap_sandbox_dbg': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -707,7 +707,7 @@
       {'name': 'v8testing', 'shards': 14},
     ],
   },
-  'v8_linux_arm64_rel_ng_triggered': {
+  'v8_linux_arm64_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -719,7 +719,7 @@
       {'name': 'v8testing', 'variant': 'extra', 'shards': 14},
     ],
   },
-  'v8_linux_arm64_cfi_rel_ng_triggered': {
+  'v8_linux_arm64_cfi_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -728,7 +728,7 @@
       {'name': 'v8testing', 'shards': 4},
     ],
   },
-  'v8_linux64_arm64_no_pointer_compression_rel_ng_triggered': {
+  'v8_linux64_arm64_no_pointer_compression_rel': {
     'swarming_dimensions' : {
       'os': 'Ubuntu-18.04',
     },
@@ -738,7 +738,7 @@
   },
   ##############################################################################
   # Linux64 with Loongson simulators
-  'v8_linux64_loong64_rel_ng_triggered': {
+  'v8_linux64_loong64_rel': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
     },
@@ -748,7 +748,7 @@
   },
   ##############################################################################
   # Linux with RISC-V simulators
-  'v8_linux_riscv32_rel_ng_triggered': {
+  'v8_linux_riscv32_rel': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
     },
@@ -756,7 +756,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_linux64_riscv64_rel_ng_triggered': {
+  'v8_linux64_riscv64_rel': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
     },
@@ -766,7 +766,7 @@
   },
   ##############################################################################
   # Odroids with native arm
-  'v8_odroid_arm_rel_ng_triggered': {
+  'v8_odroid_arm_rel': {
     'swarming_dimensions' : {
       'cores': '8',
       'cpu': 'armv7l-32-ODROID-XU4',
@@ -784,7 +784,7 @@
   },
   ##############################################################################
   # Win32
-  'v8_win_dbg_ng_triggered': {
+  'v8_win_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Windows-7-SP1',
@@ -794,7 +794,7 @@
       {'name': 'v8testing', 'shards': 3},
     ],
   },
-  'v8_win_rel_ng_triggered': {
+  'v8_win_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Windows-7-SP1',
@@ -807,7 +807,7 @@
   },
   ##############################################################################
   # Win64
-  'v8_win64_asan_rel_ng_triggered': {
+  'v8_win64_asan_rel': {
     'swarming_dimensions' : {
       'os': 'Windows-10-19042',
     },
@@ -815,7 +815,7 @@
       {'name': 'v8testing', 'shards': 5},
     ],
   },
-  'v8_win64_dbg_ng_triggered': {
+  'v8_win64_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Windows-10-19042',
@@ -827,7 +827,7 @@
       {'name': 'v8testing', 'variant': 'extra', 'shards': 2},
     ],
   },
-  'v8_win64_msvc_rel_ng_triggered': {
+  'v8_win64_msvc_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Windows-10-19042',
@@ -838,7 +838,7 @@
       {'name': 'v8testing', 'shards': 2},
     ],
   },
-  'v8_win64_rel_ng_triggered': {
+  'v8_win64_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Windows-10-19042',
@@ -852,7 +852,7 @@
   },
   ##############################################################################
   # Mac64
-  'v8_mac64_asan_rel_ng_triggered': {
+  'v8_mac64_asan_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -861,7 +861,7 @@
       {'name': 'v8testing', 'shards': 8},
     ],
   },
-  'v8_mac64_dbg_ng_triggered': {
+  'v8_mac64_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -873,7 +873,7 @@
       {'name': 'v8testing', 'variant': 'extra', 'shards': 6},
     ],
   },
-  'v8_mac64_gc_stress_dbg_ng_triggered': {
+  'v8_mac64_gc_stress_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -882,7 +882,7 @@
       {'name': 'd8testing', 'test_args': ['--gc-stress'], 'shards': 6},
     ],
   },
-  'v8_mac64_rel_ng_triggered': {
+  'v8_mac64_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -894,7 +894,7 @@
       {'name': 'v8testing', 'variant': 'extra', 'shards': 3},
     ],
   },
-  'v8_mac_arm64_rel_ng_triggered': {
+  'v8_mac_arm64_rel': {
     'swarming_dimensions' : {
       'cpu': 'arm64',
       'os': 'Mac-11',
@@ -904,7 +904,7 @@
       {'name': 'v8testing'},
     ],
   },
-  'v8_mac_arm64_dbg_ng_triggered': {
+  'v8_mac_arm64_dbg': {
     'swarming_dimensions' : {
       'cpu': 'arm64',
       'os': 'Mac-11',
@@ -914,7 +914,7 @@
       {'name': 'v8testing'},
     ],
   },
-  'v8_mac_arm64_full_dbg_ng_triggered': {
+  'v8_mac_arm64_full_dbg': {
     'swarming_dimensions' : {
       'cpu': 'arm64',
       'os': 'Mac-11',
@@ -924,7 +924,7 @@
       {'name': 'v8testing'},
     ],
   },
-  'v8_mac_arm64_no_pointer_compression_dbg_ng_triggered': {
+  'v8_mac_arm64_no_pointer_compression_dbg': {
     'swarming_dimensions' : {
       'cpu': 'arm64',
       'os': 'Mac-11',
@@ -934,7 +934,7 @@
       {'name': 'v8testing'},
     ],
   },
-  'v8_mac_arm64_sim_rel_ng_triggered': {
+  'v8_mac_arm64_sim_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -944,7 +944,7 @@
       {'name': 'v8testing', 'variant': 'future', 'shards': 2},
     ],
   },
-  'v8_mac_arm64_sim_dbg_ng_triggered': {
+  'v8_mac_arm64_sim_dbg': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -954,7 +954,7 @@
       {'name': 'v8testing', 'variant': 'future', 'shards': 2},
     ],
   },
-  'v8_mac_arm64_sim_nodcheck_rel_ng_triggered': {
+  'v8_mac_arm64_sim_nodcheck_rel': {
     'swarming_dimensions' : {
       'cpu': 'x86-64',
       'os': 'Mac-10.15',
@@ -2349,7 +2349,7 @@
       },
     ],
   },
-  'v8_numfuzz_rel_ng_triggered': {
+  'v8_numfuzz_rel': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
     },
@@ -2366,7 +2366,7 @@
       },
     ],
   },
-  'v8_numfuzz_tsan_rel_ng_triggered': {
+  'v8_numfuzz_tsan_rel': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
     },
@@ -2419,7 +2419,7 @@
       },
     ],
   },
-  'v8_numfuzz_dbg_ng_triggered': {
+  'v8_numfuzz_dbg': {
     'swarming_dimensions': {
       'os': 'Ubuntu-18.04',
     },
-- 
2.35.1

